### PR TITLE
Refer <gcAllowVeryLargeObjects> element

### PR DESCRIPTION
**Pull request by DiegoQueiroz**
The current version miss to tell what element should be changed to change the array limit

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 1]
```diff
@@ -91,7 +91,7 @@
   
  In addition to the listed set operations, the <xref:System.Collections.Generic.HashSet%601> class also provides methods for determining set equality, overlap of sets, and whether a set is a subset or superset of another set.  
   
- For very large <xref:System.Collections.Generic.HashSet%601> objects, you can increase the maximum capacity to 2 billion elements on a 64-bit system by setting the `enabled` attribute of the  configuration element to `true` in the run-time environment.  
+ For very large <xref:System.Collections.Generic.HashSet%601> objects, you can increase the maximum capacity to 2 billion elements on a 64-bit system by setting the `enabled` attribute of the configuration element `<gcAllowVeryLargeObjects>` to `true` in the run-time environment.
```
[the remark this answers is not in the document]
Comment (DiegoQueiroz) [reply xml/System.Collections.Generic/HashSet`1.xml]: Actually, this option does exist in .NET Core as well (also, it seems to be enabled by default). See https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/clr-configuration-knobs.md#environmentregistry-configuration-knobs https://github.com/dotnet/coreclr/issues/8847 https://github.com/dotnet/coreclr/pull/8853

[the remark this answers is not in the document]
Comment (mikedn) [reply xml/System.Collections.Generic/HashSet`1.xml]: Yes, it exists but it's not exposed via an XML configuration file like `<gcAllowVeryLargeObjects>` so the text might be misleading in .NET Core. And yes, it's enabled by default, I know :smile:

**Comment by BillWagner** [comment]
@safern Can you review this change? It LGTM, but I'd like you to make sure it's correct.

File: xml/System.Collections.Generic/HashSet`1.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (rpetrusha) [reply xml/System.Collections.Generic/HashSet`1.xml]: - Begin the paragraph with "\*\*.NET Framework only:\*\*
- configuration element `<gcAllowVeryLargeObjects>` --> \[\`\<gcAllowVeryLargeObjects>\`\}(~/docs/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element.md) configuration element

**Comment by Thieum** [comment]
It seems to impact other containers as well, as seen in #1282

**Comment by DiegoQueiroz** [comment]
I'm not good with PR, but is anything missing? It seems @rpetrusha request change was not addressed, but it actually is.

**Comment by rpetrusha** [comment]
No, nothing is missing, @DiegoQueiroz. Your changes look good, so I'll merge your PR now. The changes should appear on docs.microsoft.com in the next day or two.

**Pull request merged**